skore/src/skore/sklearn/_plot/data/table_report.py

```diff
@@ -0,0 +1,49 @@
+from skrub._reporting._html import to_html
+from skrub._reporting._summarize import summarize_dataframe
+
+from skore.sklearn._plot.base import Display
+
+
+class TableReportDisplay(Display):
+    def __init__(self, summary, column_filters=None):
+        self.summary = summary
+        self.column_filters = column_filters
+
+    @classmethod
```

Comment:
Let's keep the factory private. For consistency, let's call it `_compute_data_for_display`